app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -120,40 +185,98 @@ class SessionsViewPagerAdapter(
 
     private val tabs = arrayListOf<Tab>()
     private var roomTabs = mutableListOf<Tab.RoomTab>()
+    private var schedulesTabs = mutableListOf<Tab.TimeTab>()
 
-    sealed class Tab(val title: String, val fragment: Fragment, val screenName: String) {
```

Comment:
Fragment shouldn't be cached by this class.

- All is singleton
- Other tabs are not singleton

As you can see, these are belong to different lifecycles.